qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -1,6 +1,8 @@
 - pci_bridge: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     login_timeout = 360
+    dd_bs = 1K
+    dd_count = 1000
```

Comment:
Should not have this change if you had the plan to support more than block tests.